### PR TITLE
Create new Machine Tokens doc

### PR DESCRIPTION
**Pull request by nataliejeremy**
This PR adds documentation on machine tokens and updates the Users doc to add machine tokens to the list of options.
@bmackinney Please review.

### CODE REVIEW

File: source/docs/articles/local/cli/machine-tokens.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+For security purposes, we recommend removing tokens from your account when they are no longer used.   
+
+1. From your User Dashboard, click **Account**, and select [Machine Tokens](https://dashboard.pantheon.io/users/#account/tokens/).
+2. Locate the token you want to delete, and click **Revoke Token**.
+3. Type **Revoke**, and click **I understand the consequences. Revoke this token.**
+
+## Benefits of Using Machine Tokens
+
+### Continuous Integration (CI)
+Create a bot user, add a machine token, and save that machine token in an environment variable on Circle CI. Include bash scripts in your repository at `~/code/private/scripts/` that authenticate to the site with Terminus to automate repetitive procedures.
```
Comment (ari-gold) [inline source/docs/articles/local/cli/machine-tokens.md]: Circle CI is one continuous integration service of many

File: source/docs/articles/local/cli/machine-tokens.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Terminus Authentication for SAML/Single-Sign On (SSO)
+Users in Organizations with SSO can now authenticate via Terminus. You’ll no longer be redirected to the single-sign on page.
+
+
+## See Also
+- [The Ins and Outs of Token-Based Authentication](https://scotch.io/tutorials/the-ins-and-outs-of-token-based-authentication)
+- [Heroku CLI Authentication](https://devcenter.heroku.com/articles/authentication)
```
Comment (ari-gold) [inline source/docs/articles/local/cli/machine-tokens.md]: Why are we linking to Heroku? Suggest removing

**Comment by joshkoenig** [comment]
@ronan is deploying some UX tweaks. Can we add a screenshot once that's done?

File: source/docs/articles/local/cli/machine-tokens.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: Creating and Revoking Machine Tokens
+description: Learn how to create a machine token in order to use Terminus.
+---
+
+Machine tokens are used to uniquely identify your machine and securely authenticate via Terminus (as of the 0.11.0 release).
```
Comment (joshkoenig) [inline source/docs/articles/local/cli/machine-tokens.md]: Right now we should recommend that people use master for this. 

Let's put this in an alert box. Once the release is cut we can update.

**Pull request merged**